File: ulohy/23-krouzky/reseni/main.py
if __name__ == "__main__":
    m = int(input())
    input()

    math_club_members = []
    for _ in range(m):
        math_club_members.append(input())

    input()

    b = int(input())

    input()

    biology_club_members = []
    for _ in range(b):
        biology_club_members.append(input())

    for i in range(m):
        for j in range(b):
            if math_club_members[i] == biology_club_members[j]:
                print(math_club_members[i])

File: ulohy/23-krouzky/reseni/main.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});

let m;
const mathClubMembers = [];
let b;
const biologyClubMembers = [];

rl.on("line", (line) => {
    if (m === undefined) {
        m = parseInt(line);
        return;
    }

    if (line === "") {
        return;
    }

    if (mathClubMembers.length < m) {
        mathClubMembers.push(line);
        return;
    }

    if (b === undefined) {
        b = parseInt(line);
        return;
    }

    biologyClubMembers.push(line);

    if (biologyClubMembers.length < b) {
        return;
    }

    for (let i = 0; i < m; i++) {
        for (let j = 0; j < b; j++) {
            if (mathClubMembers[i] === biologyClubMembers[j]) {
                console.log(mathClubMembers[i]);
            }
        }
    }
    rl.close();
});
